evaluator/src/main/kotlin/ProjectSourceRule.kt

```diff
@@ -36,6 +37,24 @@ open class ProjectSourceRule(
      * clone and may take a while.
      */
     val projectSourcesDir: File by lazy { projectSourceTreeResolver.getRootDir() }
+
+    /**
+     * Return all files from the project's source tree which match any of the provided [glob expressions][patterns].
+     */
+    fun projectSourceFindFiles(vararg patterns: String): List<File> =
```

Comment:
Is this function supposed to be public?

Reply:
Yes: The use cases are

1. Use it from the policy rule directly (instead of the matcher)
2. Write custom matchers, re-using logic from ORT. Not that any code implemented in `Matcher.matches()` is 
    not really re-usable, so I minimize it. 

For 1, note that when implementing a policy rule often times using the matchers with their binary result is ok. But often times when one wants to write good error messages or how-to-fix, the binary result is not sufficient anymore.